src/main/java/ua/lviv/iot/manager/ShopManager.java

```diff
@@ -0,0 +1,50 @@
+package ua.lviv.iot.manager;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import ua.lviv.iot.models.loom.Loom;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+@AllArgsConstructor
+public class ShopManager {
+
+    private List<Loom> looms;
+
+    public List<Loom> searchBy(String materialOfTheProducedFabric, int widthOfTheFormedTissue) {
```

Comment:
search by what?

Reply:
By parameters in the parentheses. If I write the full name, it will take almost the whole line. Should I anyway rename it?

Reply:
naming is ok